docs/standard-library/diags/triple-quote-indent.md

```diff
@@ -0,0 +1,25 @@
+---
+title: triple-quote-indent
+---
+
+Triple quoted strings must all be at least indented to the same level as closing `"""`.
+
+#### ❌ Incorrect
+
+```tsp
+const a = """
```

Comment:
nit: can we want to have auto-format as well? Can track separately from this PR.

Reply:
the formatter will already auto format triple quotes to the current alignement but here this error is that they havae an invalid triple quote, we can't really fix it for them. Fixing it might actually cause them to lo get a different indenting that they meant.

Reply:
I'll file an issue to see if we can provide a codefix for it